WFD/swb_import.py

```diff
@@ -120,15 +120,19 @@ def process_single_swb(self, data, item):
         :param item: Wikidata item associated with a swb, or None if one
             should be created.
         """
+        bad_names = ('not applicable', )  # known (lower case) non-names
         item = item or self.create_new_swb_item(data)
         item.exists()  # load the item contents
 
         # Determine claims
         name = data.get('surfaceWaterBodyName')  # in English per page 33
+        if name and name.lower() in bad_names:
+            name = None
         protoclaims = self.make_protoclaims(data)
 
         # Upload claims
-        self.wd.addLabelOrAlias('en', name, item)
+        if name:
+            self.wd.addLabelOrAlias('en', name, item)
```

Comment:
It shouldn't (hence the "not applicable" in the Swedish data). But I know for a fact that some countries do. Now this may be because they consider their local names to also be the English names... who knows. But by design the name in the field should always be English.